styles/styles.css

```
@@ -0,0 +1,610 @@
+    font-size: 30px;
+    line-height: 36px;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+
+
+
+.contacts-phone {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 40px;
+    color: #000000;
+    text-decoration: none;
+    margin-top: 45px;
+    margin-left: 448,5px;
+    margin-right: 420,5px;
+    margin-bottom: 35px;
+}
+
+.catalog-background-header {
+    background-image: url(../images/catalog-background-1-redactor.jpg);
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: 0%;
+    color: #FFFFFF;
+}
+
+.catalog-title {
+    font-family: 'PT Sans';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 60px;
+    line-height: 78px;
+}
+
+.catalog-link {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 21px;
+}
+
+.catalog-filter-infrastructure,
+.catalog-type-of-hotels {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    text-transform: capitalize;
+}
+
+.infrastructure-list,
+.type-hotel {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 23px;
+}
+
+
+.button-official-site {
+    text-decoration: none;
+    color:#000000;
+    max-width: 115px;
+    margin-left: auto;
+}
+
+.field {
+    font-family: inherit;
+}
+
+h2 {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+    display: flex;
+    justify-content: center;
+    margin-top: 69px;
+    margin-left: 290px;
+    margin-bottom: 120px;
+    margin-right: 290px;
+    padding: 0;
+}
+
+p {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #333333;
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin-top: 25px;
+    margin-left: 274,5px;
+    margin-bottom: 69px;
+    margin-right: 274,5px;
+}
+
+
+h3 {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+.reservations {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+.reservations-text {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 26px;
+    text-align: center;
+    color: #333333;
+    text-transform: none;
+}
+
+.reservations-button {
+    font-family: inherit;
+    text-transform: uppercase;
+}
+
+.visually-hidden {
+    visibility: hidden;
+}
```

```suggestion
.visually-hidden {

}
```
Это недоступное скрытие

Можно так:
```css
.visually-hidden {
  position: fixed;
  transform: scale(0);
}
```